test/tokens.js

```diff
@@ -257,6 +258,19 @@ describe('Tokens smart contract', function () {
       const block1 = res;
       const transactionsBlock1 = block1.transactions;
 
+      console.log(JSON.parse(transactionsBlock1[2].logs).errors[0]);
```

Comment:
Clear debugging?

Reply:
I leave console logging in test cases on purpose, I like seeing what is being output while tests are running.